README.md

```diff
@@ -1,6 +1,33 @@
 # TOVE: Transcription Object Viewer/Editor
 
-*Readme and docs coming soon*	
+Tove is the API for the [ALICE Text Editor](https://github.com/zooniverse/text-editor) tool. The text editor tool takes in reductions on transcription data from Caesar, and allows users to review, update and approve transcriptions.
+
+## Setup
+
+### Run app locally
+
+1. Install dependencies:
+- Rails 6.0.1
+- Ruby 2.6.5
+- PostgreSQL 9.5
+
+2. Clone the repo `git clone https://github.com/zooniverse/tove.git` 
+
+3. `cd` into the cloned folder   
+
+4. Run `bundle install`  
+
+5. Run the following commands to set up the database and generate test data:
+```
+rake db:setup
+rake db:migrate
+rake db:seed
+```
+
+6. Run `rails s` to start the app locally
```

Comment:
not blocking: OOI `rails s` in older rails versions didn't repsect the puma config, you had to manually call that via `bundle exec puma -C config/puma.rb` (hence the start script to boot puma), https://github.com/puma/puma/issues/1253

Seems like this is fixed in newer versions of rails, is this the case?

Reply:
Was unaware that this was the case. Let me look into that.

Reply:
You are right, [it looks like it's been fixed](https://www.schneems.com/blogs/container_ready_rails_5):
> In addition to shipping with Puma, Rails also generates config/puma.rb and efforts were made to allow Puma to read this config file when it’s booted by the $ rails server command. This feature is baked into Puma 3.x+, which allows Rails to configure Puma around the number of threads being generated.